Jenkinsfile

```diff
@@ -0,0 +1,62 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+stage('Trust') {
+    enforceTrustedApproval('aerogear')
+}
+
+def prLabels = getPullRequestLabels {}
+
+def androidSdkSlave = containerTemplate(
+  name: 'jnlp', 
+  image: 'docker.io/aerogear/digger-android-slave-image:latest',
+  args: '${computer.jnlpmac} ${computer.name}',
+  ttyEnabled: false)
+
+def runIntegrationTest() {
+  def repositoryName = "android-sdk"
+  def projectName = "test-${repositoryName}-${currentBuild.number}-${currentBuild.startTimeInMillis}"
+  def metricsApiTemplatePath = "./ci/aerogear-app-metrics.yml"
+  
+    openshift.withCluster() {
+      openshift.newProject(projectName)
+      openshift.withProject(projectName) {
+          openshift.newApp( "--file", "${metricsApiTemplatePath}" )
+          // Get URL of deployed metricsApi
+          def routeSelector = openshift.selector("route", "aerogear-app-metrics")
+          def metricsApiHost = "http://" + routeSelector.object().spec.host
+          def servicesConfigJsonPath = "./core/src/debug/assets/integration-test-mobile-services.json"
```

Comment:
@psturc @danielpassos 
Heads up, the folder where mobile-services JSON files is going to change from `debug` to `test` #167

Reply:
thanks for heads up! fixed